### PR TITLE
Add methods for string arrays

### PR DESCRIPTION
**Pull request by f19ps**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+**This branch targets Ruby 2.4+ only**
+
+
+## Usage
+
+#### Install gem:
+
+```ruby
+gem 'array_attributes'
```
Comment (juan-apa) [inline README.md:18]: isn't it `gem install array attributes`? if you are referring it to the gemfile, I would specify that. Like:
```markdown
Add this line to your  Gemfile:
gem 'array_attributes'
then run:
bundle install
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+```ruby
+book = Book.new
+book.authors_raw="John Doe, Jane Doe"
+book.save!
+book.authors #=> ["John Doe", "Jane Doe"]
+```
+
+### compact_attribute
+
+You can invoke `compact_attribute` to remove blank items from the array attribute.
```
Comment (juan-apa) [inline README.md:69]: Im wondering if this method could be omited by using a strategy similar to the one that uses nested_attributes, say: pass an aditional hash param to the `string_array_attributes(attrs, opts)` and in the opts, add a `refect_blank: true`. then inside your string_array_attributes, you could imitate the behavior, or even call the  compact_value. 

You can go a step further and allow passing a lambda, for further generalisation. Something like:
`reject_if: -> (attr){ attr.length > 5 }`

Comment (f19ps) [reply README.md:69]: This is a great idea

File: README.md (hunk 1 above)
Comment (f19ps) [reply README.md:18]: Yes, you're right

**Review by felipekb — APPROVED** [review]
LGTM

**Review by juan-apa — APPROVED** [review]
are you planning on adding tests?

**Comment by f19ps** [comment]
> 

Yes!

**Pull request merged**